### ISSUE TITLE
Accessibility: focus indicator does not consistently return to top of page

### ISSUE THREAD

**Issue by samglover** [SuffolkLITLab/docassemble-MAPetitionToSealEviction#255]
From MATC:

> When activating the next button to navigate between steps, the focus indicator often ends up remaining in the middle of the page and the change of page is not announced by the screen reader. This issue occurs with most of the steps.
> 
> The focus should return to the top of the page each time a new page loads and the page title should be announced.
